### PR TITLE
Feature/update connect client

### PR DESCRIPTION
**Pull request by mtisserant**
- update connect-client 3.1.0
- update ssh link with https link for connect-client (Travis Ci can't cloning public repository with ssh method. https link is needed)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "repositories" : [
         {
             "type" : "git",
-            "url": "git@github.com:flash-global/connect-client.git"
+            "url": "https://github.com/flash-global/connect-client.git"
```
Comment (skrijeljhasib) [inline composer.json]: Why did you changed ssh to https ?

Comment (mtisserant) [reply composer.json]: Travis Ci can't cloning public repository with ssh method. https link is needed

Comment (mtisserant) [reply composer.json]: https://travis-ci.org/flash-global/connect-package/builds/433995441?utm_source=github_status&utm_medium=notification

**Comment by fanshan** [comment]
/!\ Attention /!\

The branche `master` of this package is NOT compatible with Objective-PHP v1 but works with Objective-PHP v2.

Please consider the branche `v2` (yes the branch naming sucks) for a Objective-PHP v1 compatibility.

**Pull request closed without merging**